test/pilosa.go

```diff
@@ -263,10 +263,15 @@ func (m *Main) RecalculateCaches() error {
 
 // MustDo executes http.Do() with an http.NewRequest(). Panic on error.
 func MustDo(method, urlStr string, body string) *httpResponse {
-	req, err := gohttp.NewRequest(method, urlStr, strings.NewReader(body))
-	if err != nil {
-		panic(err)
-	}
+	req, err := gohttp.NewRequest(
+		method,
+		urlStr,
+		strings.NewReader(body),
+	)
+
+	req.Header.Set("Content-Type", "application/json")
```

Comment:
is MustDo always used for json requests? maybe we should not set headers in here at all.

Reply:
you don't have access to headers outside of the `MustDo` function.  its not that big a deal now that the Accept is not required.  I can remove if you like, i don't see any value either way

Reply:
all good